src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,8 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+    /**
+     * @return Returns a concatenated version of the printable strings of each object.
```

Comment:
Incorrect header comment. The behavior described is specific to only one implementation of the method.